ntoskrnl/include/internal/amd64/ke.h

```diff
@@ -494,6 +496,11 @@ KiProcessorFreezeHandler(
     _In_ PKTRAP_FRAME TrapFrame,
     _In_ PKEXCEPTION_FRAME ExceptionFrame);
 
+VOID
```

Comment:
```suggestion
CODE_SEG("INIT")
VOID
```

Reply:
That doesn't do anything here

Reply:
This is actually required by MSVC. GCC follows the opposite approach and doesn't care about section attributes in function prototypes.

Reply:
This is built with MSVC without changing anything:
![grafik](https://github.com/user-attachments/assets/1e012ca0-d1a6-43d7-9cf6-769c56dfb6d2)

Reply:
I completely forgot that MSVC 2015 was removed from the CI, my bad. (I expected to get a compiler error C2983)